templates/web/thamesmead/report/new/roads_message.html

```diff
@@ -0,0 +1,10 @@
+<div id="js-roads-responsibility" class="box-warning hidden">
+    <strong>Not maintained by Shropshire Council</strong>
+    <div id="js-not-a-road" class="hidden js-responsibility-message js-roads-shropshire">
+        <p>The location you have selected doesn't appear to be on <span class="js-roads-asset" data-original="a road">a public road</span> that Shropshire Council maintain.</p>
```

Comment:
Cobrand name (so Thamesmead won't be Thamesmead council - it will just be Thamesmead)